app/Http/Controllers/ProfileController.php

```diff
@@ -10,12 +10,14 @@
 class ProfileController extends Controller
 {
     // Gets profile update page
+
```

Comment:
I'll resolve that. Although, I didn't work on the profile controller.

On Mon, Nov 23, 2015 at 12:07 PM, Prosper Otemuyiwa <
notifications@github.com> wrote:

> In app/Http/Controllers/ProfileController.php
> <https://github.com/andela/pibbble/pull/141#discussion_r45591607>:
>
> > @@ -10,12 +10,14 @@
> >  class ProfileController extends Controller
> >  {
> >      // Gets profile update page
> > +
>
> It should be a multiline comment and there is no need for space between
> the comments and the method. Take care of that for everything in this file
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/andela/pibbble/pull/141/files#r45591607>.
>



-- 
*Opeyemi Ogunjimi | Software Developer | Andela*
(c) +234-703-936-2475 | Andela.com <http://andela.com/>